frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/FeatureOverviewEnvironment.tsx

```diff
@@ -73,12 +75,38 @@ export const FeatureOverviewEnvironment = ({
     const projectId = useRequiredPathParam('projectId');
     const featureId = useRequiredPathParam('featureId');
     const { isOss } = useUiConfig();
+    const envAddStrategySuggestionEnabled = useUiFlag(
+        'envAddStrategySuggestion',
+    );
     const hasActivations = Boolean(
         environment?.enabled ||
             (environment?.strategies && environment?.strategies.length > 0) ||
             (environment?.releasePlans && environment?.releasePlans.length > 0),
     );
 
+    if (!hasActivations && envAddStrategySuggestionEnabled) {
+        return (
+            <StyledFeatureOverviewEnvironment>
+                <EmptyEnvironmentSection
+                    environmentId={environment.name}
+                    projectId={projectId}
+                    featureId={featureId}
+                >
+                    <FeatureOverviewEnvironmentToggle
+                        environment={environment}
+                    />
+                    <FeatureStrategyMenu
+                        label='Add strategy'
+                        projectId={projectId}
+                        featureId={featureId}
+                        environmentId={environment.name}
+                        variant='outlined'
+                    />
+                </EmptyEnvironmentSection>
+            </StyledFeatureOverviewEnvironment>
+        );
+    }
```

Comment:
Was this easier to implement than re-using the current component below?

Reply:
Yes. At least while it's still a feature flag, didn't want to make too many structural changes to the existing component. Rhe original plan was to make a parallel component and then merge them together when removing the flag. I can make another pass at that however, if you think it's the wrong way to go about it.

Reply:
No, I'm okay with this, was just curious.